# bu dosyanın ismini değiştirerek bir IOError oluşturabilirsiniz (exception3.py)